cf/requirements.txt

```diff
@@ -24,6 +24,6 @@ whitenoise==2.0.6
 #imposm
 numpy==1.11.1
 -e git://github.com/terranodo/osm-export-tool2.git@cf-tests#egg=oet2
--e git://github.com/terranodo/mapproxy.git@addGeopackageAutoconfig#egg=mapproxy
--e git://github.com/terranodo/django-mapproxy.git@master#egg=djmp
```

Comment:
👏  this makes me happy. 😄 

Does this mean that `eventkit-cloud` doesn't need the guardian permissions framework enabled?